### DISCUSSION TITLE
Add Slides from another file

### DISCUSSION THREAD

**Discussion by Sarah-Thomsen** [hakimel/reveal.js#3177] [Q&A]
Let's say i have two reveal.js presentations, one with many slides and one with only a few slides. Is it possible to add the slides from one presentation too another using code? I'm aware I could copy and paste the sections but I am searching for another possible way. 
I thought of adding code too the presentation with many slides like <section "Attribut telling to use the slides from presentation 2"></section> or anything like that. Is that possible?

**Comment by datavectors** [comment]
I have just replied to an idea here in support of modularity of slides.
https://github.com/hakimel/reveal.js/discussions/3064

You can use the same approach to "merge" slides into a mainstream slides show.

https://www.php.net/manual/en/function.require.php

Just ensure that you "require" the block of slides as slides.html and not the surrounding reveal.js code.

**Comment by peter-lyons-kehl** [comment]
If you are happy with writing content in Markdown (supported by VS Code and other editors/extensions, GitHub...), then you can load ["External" Markdown](https://revealjs.com/markdown/#external-markdown) files - whether from the same directory, upper/sub directory on the same domain, or a different domain.
